oamapps/mcsadmin/mcsadmin.cpp

```diff
@@ -7841,24 +7864,17 @@ int processCommand(string* arguments)
                     {
                         try
                         {
-							cout << endl << "   Restarting System " << endl;
-							gracefulTemp = oam::FORCEFUL;
-							int returnStatus = oam.restartSystem(gracefulTemp, ackTemp);
-							switch (returnStatus)
-							{ 
-								case API_SUCCESS:
-									if ( waitForActive() )
-										cout << endl << "   Successful restart of System " << endl << endl;
-									else
-										cout << endl << "**** restartSystem Failed : check log files" << endl;
-									break;
-								case API_CANCELLED:
-									cout << endl << "   Restart of System canceled" << endl << endl;
-									break;
-								default:
-									cout << endl << "**** restartSystem Failed : Check system logs" << endl;
-									break;
-							}
+
```

Comment:
this new functionality doesn't seem to match what was deleted.  It looks like the old version would restart the system, and the new one starts modules.  Is that intentional?